ccm_web/server/src/api/api.controller.ts

```diff
@@ -77,4 +103,26 @@ export class APIController {
     if (isAPIErrorData(result)) throw new HttpException(result, result.statusCode)
     return result
   }
+
+  @UseInterceptors(InvalidTokenInterceptor)
+  @ApiSecurity('CSRF-Token')
+  @Post('sections/:id/merge/:course')
+  async mergeSection (
+    @Param('id', ParseIntPipe) sectionId: number,
+      @Param('course', ParseIntPipe) courseId: number,
+      @UserDec() user: User
+  ): Promise<CanvasCourseSectionBase> {
+    const result = await this.apiService.mergeSection(user, sectionId, courseId)
+    if (isAPIErrorData(result)) throw new HttpException(result, result.statusCode)
+    return result
+  }
+
+  @UseInterceptors(InvalidTokenInterceptor)
+  @ApiSecurity('CSRF-Token')
+  @Delete('sections/:id/unmerge')
```

Comment:
Pretty sure there is no unmerge requirement for CCM, was this  just out of completeness?

Reply:
It's listed in issue #16, and I see it in the admin version wireframes ("unmerge" button on success screen?).

Reply:
You're right.. I was remembering talking about no umerging in a zoom call but now I can't remember the whole context.  Carry on :)